modules/configuration/Configuration.py

```diff
@@ -0,0 +1,16 @@
+from modules.configuration.ArgumentsConfig import ArgumentsConfig
+from modules.configuration.EnvironmentConfig import EnvironmentConfig
+from modules.configuration.YamlConfig import YamlConfig
+
+
+class Configuration(YamlConfig, ArgumentsConfig, EnvironmentConfig):

```

Comment:
Don't let Configuration inherit, do it the other way round.
1. Let Configuration be the BaseClass, all the three config classes are inheriting. 
2. implement an @abstractmethod `get_config`, which is empty in `Configration`, which every class has to implement. Move your functionality from init to get_config. Follow below comment in exporter.py